docs/data/joy/guides/using-joy-ui-and-material-ui/using-joy-ui-and-material-ui.md

```diff
@@ -130,23 +92,17 @@ Visit the following CodeSandbox to preview this use case setup.
 
 ## Case B: Material UI in a Joy UI project
```

Comment:
Do we really need these two cases still? In any of the use-cases one of the libraries can be scoped and that's it, unless I am missing something.

Reply:
Good point, both sections are combined.